dvc/repo/experiments/executor/base.py

```diff
@@ -709,6 +713,7 @@ def _repro_args(cls, dvc):
         args_path = os.path.join(dvc.tmp_dir, cls.PACKED_ARGS_FILE)
         if os.path.exists(args_path):
             args, kwargs = cls.unpack_repro_args(args_path)
+            # studio config options not expected for stage repro
```

Comment:
Not sure I follow this comment

Reply:
It referred to an outdated change. I'll drop it.